metapackages/auto-instrumentations-node/README.md

```diff
@@ -13,7 +19,68 @@ Compatible with OpenTelemetry JS API and SDK `1.0+`.
 npm install --save @opentelemetry/auto-instrumentations-node
 ```
 
-## Usage
+## Usage: Auto Instrumentation
+
+This module includes auto instrumentation for all supported instrumentations and all available data exporters.
+It provides a completely automatic, out-of-the-box experience.
+
+Enable auto instrumentation by requiring this module using the --require flag:
+
+```shell
+node --require '@opentelemetry/auto-instrumentations-node/register' app.js
+```
+
+If your Node application is encapsulated in a complex run script, you can also set it via an environment variable before running Node.
+
+```shell
+env NODE_OPTIONS="--require @opentelemetry/auto-instrumentations-node/register"
+```
+
+The auto instrumentation is highly configurable using environment variables.
+Many aspects of the auto instrumentation's behavior can be configured for your needs, such as resource detectors, exporter choice, exporter configuration, trace context propagation headers, and much more.
+
+```shell
+export OTEL_TRACES_EXPORTER="otlp"
+export OTEL_EXPORTER_OTLP_PROTOCOL="http/protobuf"
+export OTEL_EXPORTER_OTLP_COMPRESSION="gzip"
+export OTEL_EXPORTER_OTLP_TRACES_ENDPOINT="https://your-endpoint"
+export OTEL_EXPORTER_OTLP_HEADERS="x-api-key=your-api-key"
+export OTEL_EXPORTER_OTLP_TRACES_HEADERS="x-api-key=your-api-key"
+export OTEL_RESOURCE_ATTRIBUTES="service.namespace=my-namespace"
+export OTEL_RESOURCE_DETECTORS="env,host,os"
```

Comment:
We had this discussion in Python as well, you may want to stick to [language specific environment variable](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#language-specific-environment-variables) for now. There is an open spec issue to add this specific variable but it is blocked: https://github.com/open-telemetry/opentelemetry-specification/issues/2948

Reply:
Updated with a language-specific variable for now. Hopefully this will be part of the spec at some point.